packages/token-list/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "token-list",
+  "version": "1.0.0",
+  "license": "MIT",
+  "files": [
+    "src"
+  ],
+  "main": "src/index.js",
+  "scripts": {
+    "deps:check": "dependency-check --no-dev ."
+  },
+  "dependencies": {
+    "@sushiswap/default-token-list": "^18.3.0",
+    "@uniswap/default-token-list": "^2.1.0",
```

Gabriel will want these fixed 😄 

```suggestion
    "@sushiswap/default-token-list": "18.3.0",
    "@uniswap/default-token-list": "2.1.0",
```

---

Do I?

https://github.com/bloqpriv/software-development-process/blob/master/guides/JavaScript%20Dependencies.md